src/stores/auth.js

```diff
@@ -110,6 +111,13 @@ export const useAuthStore = defineStore('auth', {
         //     - `gameService.populateGame` is called
         //     - `Game.subscribe` is called
         //     - `Game.publish` is called
+
+        const gameStore = useGameStore();
+        if (!gameId && isSpectating) {
+          let { gameId } = route.params;
```

Comment:
should be const if we're not modifying this
```suggestion
          const { gameId } = route.params;
```